resources/migrations/002-create-reviews.up.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE reviews (
+       id INTEGER PRIMARY KEY,
+       title TEXT NOT NULL,
+       review_date TEXT NOT NULL,
+       CONSTRAINT title_not_empty CHECK (title <> ''),
+       CONSTRAINT date_not_empty CHECK (review_date <> '')       
+);
```

Comment:
Just want a clarification. Where are the many users associated with a review event stored? In the user-reviews table also there are only 2 user ids associated (to and from).